controller/ItemImportSampleDownload.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * DownloadSampleCsv Controller provide actions to download sample CSV
+ *
+ * @author Chinnu Francis - TAO Team - {@link http://www.tao.lu}
+ * @package taoQtiItem
+ * @license GPLv2  http://www.opensource.org/licenses/gpl-2.0.php
+ */
+
+namespace oat\taoQtiItem\controller;
+
+use oat\taoQtiItem\model\import\SampleTemplateDownload;
+use tao_actions_CommonModule;
+
+/**
+ * DownloadSampleCsv Controller provide actions to download sample CSV
+ *
+ * @author Chinnu Francis - TAO Team - {@link http://www.tao.lu}
+ * @package taoQtiItem
+
+ * @license GPLv2  http://www.opensource.org/licenses/gpl-2.0.php
+ */
```

Comment:
Please, remove this comment

https://github.com/oat-sa/extension-tao-itemqti/pull/1751/files#r642972253